### DISCUSSION TITLE
Is L3 supported?

### DISCUSSION THREAD

**Discussion by GrimmiMeloni** [jasonacox/Powerwall-Dashboard#364] [Q&A] [answered]
I noticed, that on the dashboard's frequency and voltages sections, I only see metrics for L1 and L2.
My home's installation has an L3 (I am located in Germany) that the vitals endpoint also gives data for. 
Should I expect to see that data, or is the dashboard ignoring this data point?

**Comment by jasonacox** [comment, marked as answer]
Hi @GrimmiMeloni !  I have good news. That data is being collected and is in the Grafana panel already. However it is turned off or it skews (flattens) the graphs for single-phase installations that have zero values for L3.  You will need to edit the "Frequency" panel, then scroll down through all the "L3" areas and "Unhide" them by clicking the eye:

<img width="202" alt="image" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/836718/47560153-2fb4-4604-9931-64c99e7e0bad">



<img width="651" alt="image" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/836718/08951678-0e58-4114-9311-56b97afca1f9">

Also, make sure you click the "Save" button at the stop or it will revert. :)

**Comment by GrimmiMeloni** [comment]
Excellent Jason - almost embarrassing. I went in there, and saw the areas for L3, but since they were not rendering I immediately concluded the data is not being collected. 🤦‍♂️ 

Just activated those section, and immediately saw the data.

Thanks!
